docs/changelog.md

```diff
@@ -11,6 +11,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/main/.config.yml)
 
+* **support-ktx**
+  * 🚒 Bug fixed [issue #11374](https://github.com/mozilla-mobile/android-components/issues/11374) - Restore immersive mode after interacting with other Windows. Handle this internally without clients having to pass a OnWindowFocusChangeListener.
```

Comment:
nit: I think we should mention that we have a breaking change on `enterToImmersiveMode`

Reply:
Thanks! Will update!